adapters/beachfront/beachfront.go

```diff
@@ -36,40 +42,32 @@ type BeachfrontRequests struct {
 // ---------------------------------------------------
 
 type BeachfrontVideoRequest struct {
-	IsPrebid bool                  `json:"isPrebid"`
-	AppId    string                `json:"appId"`
-	Domain   string                `json:"domain"`
-	Id       string                `json:"id"`
-	Imp      []BeachfrontVideoImp  `json:"imp"`
-	Site     BeachfrontSite        `json:"site"`
-	Device   BeachfrontVideoDevice `json:"device"`
-	User     openrtb.User          `json:"user"`
-	Cur      []string              `json:"cur"`
-}
-
-type BeachfrontSite struct {
-	Page string `json:"page"`
-}
-
-type BeachfrontPublisher struct {
-	Id string `json:"id"`
-}
-
-type BeachfrontVideoDevice struct {
-	Ua         string             `json:"ua"`
-	Devicetype int                `json:"deviceType"`
-	Geo        BeachfrontVideoGeo `json:"geo"`
-}
-
-type BeachfrontVideoGeo struct {
-	Ip string `json:"ip"`
+	IsPrebid bool   `json:"isPrebid"`
+	AppId    string `json:"appId"`
+	ID       string `json:"id"` // This ID is unique to this client page load and is sent by
+	// prebid.js. I'm sending it with banner requests as "requestId" for possible future reporting.
+	Imp    []BeachfrontVideoImp  `json:"imp"`
+	Site   openrtb.Site          `json:"site"`
+	Device BeachfrontVideoDevice `json:"device"`
+	User   openrtb.User          `json:"user"`
+	Cur    []string              `json:"cur"`
 }
 
+// Soooo close, but not quite openRTB
 type BeachfrontVideoImp struct {
 	Video    BeachfrontSize `json:"video"`
 	Bidfloor float64        `json:"bidfloor"`
-	Id       int            `json:"id"`
-	ImpId    string         `json:"impid"`
+	Id       int            `json:"id"` // A sequential count of which imp on the page this is. Since the bfm
+	// 	video endpoint only returns one response for one imp, this is
+	//	never going to happen. This will always be 0. @TODO - Alex
```

Comment:
There are a few TODOs in here... do you want us to wait until those are resolved before we merge, or did you plan to resolve them later?

If they've already been resolved, remove the TODOs?